benchmarks/Cargo.toml

```diff
@@ -29,7 +29,6 @@ rust-version = "1.70"
 [features]
 ci = []
 default = ["mimalloc"]
-simd = ["datafusion/simd"]
```

Comment:
arrow 50 removed the manual SIMD implementation and now relies on auto vectorization - https://github.com/apache/arrow-rs/pull/5184